.github/workflows/ubuntu.yml

```diff
@@ -1,76 +1,67 @@
         exclude:
           - ubuntu-version: ubuntu-18.04
-            compiler-version: gcc-10
+            compiler-version: {cc: gcc-10, cxx: g++-10}
           - ubuntu-version: ubuntu-18.04
-            compiler-version: clang-11
-    steps: # each - is a new sequentially run step
-      - name: clone-repo
-        uses: actions/checkout@v1
-
-      - name: setup-python
-        uses: actions/setup-python@v1
+            compiler-version: {cc: clang-11, cxx: clang++-11}
+    env:
+      CC: ${{ matrix.compiler-version.cc }}
+      CXX: ${{ matrix.compiler-version.cxx }}
+      CPLEX_URL: ${{ secrets.CPLEX_URL }}
+      SOPLEX_URL: ${{ secrets.SOPLEX_URL }}
+      DOWNWARD_CPLEX_ROOT: /home/runner/work/downward/lib/ibm/ILOG/CPLEX_Studio129/cplex
+      DOWNWARD_SOPLEX_ROOT: /home/runner/work/downward/lib/soplex-3.1.1
+      DOWNWARD_COIN_ROOT: /home/runner/work/downward/lib/coin
+    steps: # Each "-" is a new sequentially run step
+      - name: Clone repository
+        uses: actions/checkout@master
+
+      - name: Install Python
+        uses: actions/setup-python@master
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: setup-dependencies
+      - name: Install dependencies
         run: |
-          sudo apt-get update
-          sudo apt-get -y install zlib1g-dev libgmp3-dev
-          sudo apt-get -y install ${{ matrix.compiler-version }}
-          export CC=${{ matrix.compiler-version }}
-          if [[ $CC == gcc* ]]; then
-            export CXX="$(echo "${CC}" | sed "s/cc/++/g")";
-          elif [[ $CC == clang* ]]; then
-            export CXX="$(echo "${CC}" | sed "s/clang/clang++/g")";
-          else
-            echo "Unknown compiler version";
-            exit 1;
-          fi
+          sudo apt-get -y install zlib1g-dev libgmp3-dev ${{ matrix.compiler-version.cc }}
           mkdir /home/runner/work/downward/lib
 
-      # We only want to setup osi if both LP solvers are set, hence
+      # We only want to set up Osi if both LP solvers are set, hence
       # we execute the following three steps only if both secrets
       # are set.
-      - name: setup-cplex
-        env:
-          CPLEX_URL: ${{ secrets.CPLEX_URL }}
-          SOPLEX_URL: ${{ secrets.SOPLEX_URL }}
-          DOWNWARD_CPLEX_ROOT: /home/runner/work/downward/lib/ibm/ILOG/CPLEX_Studio129
+      - name: Install CPLEX
         if: ${{ env.CPLEX_URL != 0 && env.SOPLEX_URL != 0 }}
+        env:
+          CPLEX_STUDIO: /home/runner/work/downward/lib/ibm/ILOG/CPLEX_Studio129
         run: |
           # We redirect output of wget to hide the secret URLs.
           wget $CPLEX_URL &> /dev/null
           export CPLEX_INSTALLER=cplex_studio129.linux-x86-64.bin
           chmod +x $CPLEX_INSTALLER
-          ./$CPLEX_INSTALLER -DLICENSE_ACCEPTED=TRUE -DUSER_INSTALL_DIR=${DOWNWARD_CPLEX_ROOT} -i silent
+          ./$CPLEX_INSTALLER -DLICENSE_ACCEPTED=TRUE -DUSER_INSTALL_DIR=${CPLEX_STUDIO} -i silent
```

Comment:
Did you detect this due to the new LP solver tests?
Now we have two variables which are dependent on another. If we ever have to change one, we also have to change the other. I suggest:
1. remove CPLEX_STUDIO from the env
2. ${CPLEX_STUDIO} -> "$(dirname "${DOWNWARD_CPLEX_ROOT}")"
- if you think the line is too long or ugly, just store it in a variable, like CPLEX_STUDIO
(the amount of " symbols is correct :)

Reply:
No, I realized this when archiving libraries. I'll do the suggested change.